### PR TITLE
setHeader in RestApiContext to replace header instead of add it

### PR DESCRIPTION
**Pull request by MikaFima**

### CODE REVIEW

**Comment by shouze** [comment]
:+1:

**Comment by tyx** [comment]
We need test

File: src/RestApiContext.php [hunk 1]
```diff
@@ -113,6 +121,14 @@ public function printResponse()
     }
 
     /**
+     * @return array
+     */
+    public function getHeaders()
```
Comment (MikaFima) [inline src/RestApiContext.php]: @tyx I havn't found another way to test headers

File: features/send_request.feature [hunk 2]
```diff
@@ -22,3 +22,20 @@ Feature: Test send API request
...
         Then the JSON node "method" should be equal to "POST"
         And the JSON node "headers.content-type[0]" should be equal to "application/json"
+
+    Scenario: Add same header 2 times
+        Given I add "header" header equal to "value"
+        Given I add "header" header equal to "value2"
+        When I send a POST request to "echo"
+        Then the response status code should be 200
+        And the JSON node "headers.header" should have 1 element
+        And the JSON node "headers.header[0]" should be equal to "value, value2"
```
Comment (MikaFima) [inline features/send_request.feature]: It looks weird to have an array but all values in the same node

Comment (tyx) [inline features/send_request.feature]: It is how headers http work in fact

**Pull request merged**